frontend/public/style.scss

```diff
@@ -105,3 +105,6 @@
 @import "components/dashboard/dashboard";
 @import "components/dashboard/dashboard-card/card";
 @import "components/dashboard/health-card/health-card";
+
+// plugins
+@import "../packages/plugin-style";
```

Comment:
This shouldn't be necessary (?)

Reply:
Please note, this comes from pulled #1682 which was reworked and this stacked PR was not yet rebased.